hack/make/.ensure-frozen-images

```diff
@@ -20,7 +20,10 @@ fi
 if ! docker inspect "${images[@]}" &> /dev/null; then
 	hardCodedDir='/docker-frozen-images'
 	if [ -d "$hardCodedDir" ]; then
-		( set -x; tar -cC "$hardCodedDir" . | docker load )
+		# Do not use a subshell for the following command. Windows CI
```

Comment:
Also, when we get to Windows to Windows CI, it will go through this path (assuming we use .sh scripts rather than native PowerShell scripts).